README.md

```diff
@@ -38,6 +38,11 @@ Used versions of the IDS-testbed components:
 - Dataspace Connector (DSC) v7.1.0
 - MetadataBroker (MDB) v5.0.0
 
+## IDS Reference Testbed implementations
+| Provider      | Description     |   Components used |  Public or Private Use? |
+| ------------- | --------------- |   --------------- | ------------------------
+| ...           | ...             |   ...    | .... |
```

Comment:
| Engineering | Verifying TRUEConnector compatibility with reference implementation | DSC v7, MDB v5, Omejdn v1.6 | Private |

Reply:
@IgorBalog-Eng thanks for the input.
Can you confirm the exact versions of the components used?
- DSC 7.1.0?
- MDB v5.0.0?
- DAPS v1.6.0?

Reply:
Yes, I confirm that those are the versions from docker-compose and .env file

Reply:
Perfect, @Sonia-IDSA has just submmited the changes to the [README.md](https://github.com/International-Data-Spaces-Association/IDS-testbed/tree/Sonia-IDSA-patch-1#readme) file.
@IgorBalog-Eng could you please review and approve this pull request?
Thanks in advance